packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -105,32 +106,34 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
             <Controller
               name="role"
               control={newMemberFormMethods.control}
-              defaultValue={options[0]}
-              render={({ field: { onChange } }) => (
+              defaultValue={options[1]}
+              render={({ field: { option, onChange } }) => (
                 <div>
-                  <label
-                    className="mb-1 block text-sm font-medium tracking-wide text-gray-700"
-                    htmlFor="role">
+                  <Label className="font-medium text-gray-900" htmlFor="role">
                     {t("role")}
-                  </label>
-                  <Select
-                    defaultValue={options[0]}
-                    options={options.filter((option) => option.value !== "OWNER")}
+                  </Label>
+                  <ToggleGroup
+                    isFullWidth={true}
                     id="role"
-                    name="role"
-                    className="mt-1 block w-full rounded-sm border-gray-300 text-sm"
-                    onChange={onChange}
+                    onValueChange={onChange}
+                    defaultValue="member"
+                    options={[
+                      { value: "owner", label: t("owner") },
+                      { value: "member", label: t("member") },
+                    ]}
                   />
                 </div>
               )}
             />
             <Controller
               name="sendInviteEmail"
               control={newMemberFormMethods.control}
-              defaultValue={false}
+              defaultValue={true}
               render={() => (
-                <div className="relative flex items-start">
+                <div className="relative  flex items-start">
                   <CheckboxField
```

Comment:
Minor NIT that isn't your fault - I don't think this Field needs to be inside of a controller